packages/amplify-cli-logger/src/Redactor.ts

```diff
@@ -0,0 +1,59 @@
+const containsToRedact = ['key', 'id', 'password', 'name', 'arn', 'address', 'app', 'bucket'];
+const quotes = '\\\\?"';
+const keyMatcher = `\\w*?(${containsToRedact.join('|')})\\w*?`;
+const completeMatch = `${quotes}(${keyMatcher})${quotes}:\\s?${quotes}([^!\\\\?"]+)${quotes}`;
+
+export function Redactor(arg: string | undefined): string {
+  if (!arg) return '';
+
+  // matches any json and gives values in json
```

Comment:
Nice this is very good - When faced with complex function that is hard to read for future maintainers (which could be you), it's a good idea to give the function a more descriptive name or add a comment describing what is happening 💯